R/pkg/R/DataFrame.R

```diff
@@ -3715,3 +3715,25 @@ setMethod("rollup",
             sgd <- callJMethod(x@sdf, "rollup", jcol)
             groupedData(sgd)
           })
+
+#' alias
+#'
+#' @aliases alias,SparkDataFrame-method
+#' @rdname alias
+#' @name alias
+#' @examples
```

Comment:
add `@family SparkDataFrame functions`
I think we should probably review all these `@family` at one point...